### DISCUSSION TITLE
'bearer' is an invalid scheme for cURL.

### DISCUSSION THREAD

**Discussion by hytea** [GitbookIO/community#485] [4. Open API] [answered]
The cURL examples in your documentation are not valid. Bearer is case-sensitive as an Authorization header. 

![image](https://github.com/GitbookIO/community/assets/15982991/74753e80-3ecf-426d-8ebc-ace1b927bea3)

Should be something like this:

```
curl -L \
  -H 'Authorization: Bearer <token>' \
  'https://api.gitbook.com/v1/spaces/{spaceId}/content'
```

**Comment by addisonschultz** [comment]
Thanks for pointing this out! We'll hope on this one asap 👍

**Reply by SamyPesse** [reply, marked as answer]
It was solved, thank you!
